cypress/integration/adminFlows/pages/landingPage.spec.js

```diff
@@ -0,0 +1,145 @@
+describe('Set a landing page from the admin portal', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.fixture('users/adminUser.json').as('user');
+
+    cy.get('@user').then((user) => {
+      cy.loginUser(user).then(() => {
+        cy.visit('/admin/customization/pages');
+      });
+    });
+  });
+
+  it('should set a landing page when no other landing page exists', () => {
+    cy.findAllByRole('link', { name: 'Edit' }).first().click();
+    cy.findAllByRole('checkbox', { name: "Use as 'Locked Screen'" })
+      .first()
+      .check();
```

Comment:
```suggestion
    cy.findByRole('checkbox', { name: "Use as 'Locked Screen'" }).check();
```

suggestion (not blocking) - there's only one, so you don't necessarily have to find all 🙂